addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
viz addons make use of GUIRenderingControl just like any other addon can do. GUIRenderingControl does not belong solely to the viz. I have changes this 3 years ago.

no big deal to break api but in this case I submit this to mainline first. otherwise my branch is not compatible with addons of mainline. 

could you prepare something with the changes discussed?